apps/train_loc/lib/train_loc/encoder/vehicle_positions_enhanced.ex

```diff
@@ -59,29 +104,40 @@ defmodule TrainLoc.Encoder.VehiclePositionsEnhanced do
     %{start_date: start_date(vehicle.timestamp)}
   end
 
+  @spec entity_vehicle(Vehicle.t() | NewVehicle.t()) :: entity_vehicle_t()
   defp entity_vehicle(%{trip: "000"} = vehicle) do
     %{
       id: vehicle.vehicle_id,
       assignment_status: "unassigned"
     }
   end
 
+  defp entity_vehicle(%{trip: "N/A"} = vehicle) do
```

Comment:
suggestion: to avoid this duplication, could you update the parser to convert both `"000"` and `"N/A"` to `:unassigned`, and then treat them as a single case in here?